lib/whitehall_importer/import.rb

```diff
@@ -32,15 +40,17 @@ def call
 
         create_timeline_entry(document_import.document.current_edition)
         document_import.update!(state: "imported")
+        document_import
       end
-    rescue StandardError
-      # restore any attributes set during import
-      document_import.reload
-      raise
     end
 
   private
 
+    def export_whitehall_document
+      GdsApi.whitehall_export.document_export(document_import.whitehall_document_id)
+                             .to_h
+    end
```

Comment:
Rather than have this method response set the instance variable I wonder if we should have it as a lazily loaded variable. For example:

```
    def whitehall_document
      @whitehall_document ||= GdsApi.whitehall_export
                                    .document_export(document_import.whitehall_document_id)
                                    .to_h
    end
```

Reply:
Done